server/core/src/miscServerFunct.cpp

```diff
@@ -2976,53 +2982,28 @@ irods::error get_script_output_single_line(
     const std::string&              script_name,
     const std::vector<std::string>& args,
     std::string&                    output ) {
-    output.clear();
-    std::stringstream exec;
+    namespace bp = boost::process;
     try {
-    exec << script_language
-         << " " << irods::get_irods_home_directory().string()
-         << "/scripts/" << script_name;
-    } catch (const irods::exception& e) {
-        irods::log(e);
-        return ERROR(-1, "failed to get irods home directory");
-    }
-    for ( std::vector<std::string>::size_type i = 0; i < args.size(); ++i ) {
-        exec << " " << args[i];
-    }
-
-    FILE *fp = popen( exec.str().c_str(), "r" );
-    if ( fp == NULL ) {
-        return ERROR( SYS_FORK_ERROR, "popen() failed" );
-    }
-
-    std::vector<char> buf( 1000 );
-    const char* fgets_ret = fgets( &buf[0], buf.size(), fp );
-    if ( fgets_ret == NULL ) {
-        std::stringstream msg;
-        msg << "fgets() failed. feof["
-            << std::feof( fp )
-            << "] ferror["
-            << std::ferror( fp ) << "]";
-        const int pclose_ret = pclose( fp );
-        msg << " pclose[" << pclose_ret << "]";
-        return ERROR( FILE_READ_ERR, msg.str() );
-    }
-
-    const int pclose_ret = pclose( fp );
-    if ( pclose_ret == -1 ) {
-        return ERROR( SYS_FORK_ERROR,
-                      "pclose() failed." );
+        auto tmp = args;
+        auto resolved_program = bp::search_path(script_language);
```

Comment:
Should we make sure that this returns something? Or is it okay that this is empty?